desired_state_generator/src/gen_desired_state.py

```diff
@@ -0,0 +1,192 @@
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import hashlib
+import json
+import os
+import re
+import tempfile
+from typing import Any, Dict, List, Optional
+
+import velocitas_lib
+import velocitas_lib.services
+
+VSS_SOURCE_DEFAULT = "vss-source-default-vss"
+VSS_SOURCE_CUSTOM = "vss-source-custom-vss"
+DATABROKER_ID = "data-broker-grpc"
+GRPC_INTERFACE_ID = "dataprovider-proto-grpc"
+DATABROKER_PACKAGE_ID = "vehicledatabroker"
+MQTT_PACKAGE_ID = "mqtt-broker"
+
+VSS_INTERFACE = "vehicle-signal-interface"
```

Comment:
We should prefix these constants so we know which are used in which context:

```suggestion
VELOCITAS_IF_TYPE_VSI = "vehicle-signal-interface"
```